content/en/docs/distributions/gke/deploy/deploy-cli.md

```diff
@@ -164,6 +164,10 @@ To deploy Kubeflow, run the following command:
 make apply
 ```
 
+* If deployment returns an error due to missing resources in `serving.kserve.io` API group, rerun `make apply`. This is due to a race condition between CRD and runtime resources in KServe.
+
+  * This issue is being tracked in [kubeflow/gcp-blueprints#384](https://github.com/kubeflow/gcp-blueprints/issues/384)
+
 * If resources can't be created because `webhook.cert-manager.io` is unavailable wait and
```

Comment:
Question: Is this issue resolved now?

Reply:
Yes, this is resolved in [this PR](/kubeflow/gcp-blueprints/pull/387). However, it may be relevant for someone who uses prior versions.

Reply:
Sounds good. Thanks!